src/api/spec/helpers/webui/webui_helper_spec.rb

```diff
@@ -319,7 +319,61 @@
   end
 
   describe '#user_and_role' do
-    skip('Please add some tests')
+    let(:user) { create(:user) }
+    let(:nobody) { create(:user_nobody) }
```

Comment:
But anyway the nobody user is created when needed if it is not in the database: https://github.com/openSUSE/open-build-service/blob/master/src/api/app/models/user.rb#L292